internal/middlewere/log.go

```diff
@@ -0,0 +1,28 @@
+package middlewere
+
+import (
+	"github.com/gin-gonic/gin"
+	"go.uber.org/zap"
+	"time"
+)
+
+func RequestLogger(logger *zap.Logger) gin.HandlerFunc {
```

Comment:
```suggestion
func LoggerMiddleware(logger *zap.Logger) gin.HandlerFunc {
```
мидлвары всегда стоят между реквестом и логикой ручек